src/terrain/terrain.js

```diff
@@ -182,6 +229,7 @@ export class Terrain extends Elevation {
     _style: Style;
     orthoMatrix: mat4;
     enabled: boolean;
+    renderMode: number;
```

Comment:
Is this assigned somewhere? I might be missing something but I don't see it used.

Reply:
Yep, good catch!